code/game/objects/items/weapons/storage/nitrilebox.dm

```diff
@@ -0,0 +1,77 @@
+/obj/item/nitrilebox
+	name = "nitrile gloves box"
+	icon = 'icons/obj/storage.dmi'
+	icon_state = "nitrilebox"
+	item_state = "sheet-metal"
+	drop_sound = 'sound/items/drop/cardboardbox.ogg'
+	pickup_sound = 'sound/items/pickup/cardboardbox.ogg'
+	throwforce = 1
+	w_class = 5
+	throw_speed = 3
+	throw_range = 7
+	layer = OBJ_LAYER - 0.1
+	var/amount = 50					//How much paper is in the bin.  Gloves, actually, but this code is copied from the paper bin code.
+
+/obj/item/nitrilebox/MouseDrop(mob/user as mob)
+	if((user == usr && (!( usr.restrained() ) && (!( usr.stat ) && (usr.contents.Find(src) || in_range(src, usr))))))
+		if(!istype(usr, /mob/living/carbon/slime) && !istype(usr, /mob/living/simple_animal))
+			if( !usr.get_active_hand() )		//if active hand is empty
+				var/mob/living/carbon/human/H = user
+				var/obj/item/organ/external/temp = H.organs_by_name[BP_R_HAND]
+
+				if (H.hand)
+					temp = H.organs_by_name[BP_L_HAND]
+				if(temp && !temp.is_usable())
+					to_chat(user, "<span class='notice'>You try to move your [temp.name], but cannot!</span>")
```

Comment:
We're using SPAN macros now, these work like this:

```suggestion
					to_chat(user, SPAN_NOTICE("You try to move your [temp.name], but cannot!"))
```

They're defined for all the span classes used in this file, in _macros.dm if you want to look at them.